app/scenes/select_server/select_server.js

```diff
@@ -77,7 +79,8 @@ export default class SelectServer extends PureComponent {
     };
 
     onClick = async () => {
-        const url = this.props.serverUrl;
+        var preUrl = urlParse(this.props.serverUrl, true);
```

Comment:
Can this be changed to `const preUrl`? That way it will be consistent with the rest of codebase.